feature/achievements/src/main/java/io/github/droidkaigi/confsched2023/achievements/AchievementsScreen.kt

```diff
@@ -138,7 +140,7 @@ fun AchievementScreenDialog(
             )
         },
         text = {
-            Column {
+            Column(modifier = Modifier.verticalScroll(rememberScrollState())) {
```

Comment:
```
val scrollState = rememberScrollState()
```

When considering the reuse of scrollState, you might find it helpful to store `rememberScrollState()` in a variable, much like what's suggested in the official documentation.

Reply:
Thanks for the review! I'll fix this.

Reply:
I fixed this!
Could you kindly consider reviewing it again, if it's not too much trouble?

[store ScrollState in a variable for reusability](https://github.com/DroidKaigi/conference-app-2023/pull/1155/commits/6367c5dcecc70cbe9ecaa01c77f352cb730a8e54)